confgenerator/fluentbit/conf.go

```diff
@@ -440,9 +401,8 @@ func (s Syslog) Generate() (string, error) {
 // A WindowsEventlog represents the configuration data for fluentbit's winlog input plugin
 type WindowsEventlog struct {
 	Tag          string
-	Channels     string
-	Interval_Sec string
-	DB           string
+	Channels     string // XXX: proper type
```

Comment:
Should this be `[]string` as well?

Reply:
In theory... hence the comment. I wasn't addressing it because this type is going to be merged, but w/e, I improved it.